docs/shader-specifications/1-shaders.md

```diff
@@ -40,6 +40,9 @@ The result of this shader is this:
 
 ![simple_uv_map](https://raw.githubusercontent.com/wolfenrain/umbra/main/assets/simple_uv_map.png)
 
+> **Note**
```

Comment:
Might be worth adding this right after line 16 instead. As we mention the constraints in the section above that already